library/src/main/java/com/google/android/exoplayer2/audio/AudioTrack.java

```diff
@@ -443,6 +443,14 @@ public void configure(String mimeType, int channelCount, int sampleRate,
     }
 
     boolean passthrough = !MimeTypes.AUDIO_RAW.equals(mimeType);
+    if (Util.SDK_INT <= 23 && "foster".equals(Util.DEVICE) && "NVIDIA".equals(Util.MANUFACTURER)) {
```

Comment:
changed.  Earlier code used the passthrough variable but that didn't fully fix it.